sklearn/model_selection/_validation.py

```diff
@@ -474,6 +476,18 @@ def _fit_and_predict(estimator, X, y, train, test, verbose, fit_params,
         estimator.fit(X_train, y_train, **fit_params)
     func = getattr(estimator, method)
     predictions = func(X_test)
+    if method in ['decision_function', 'predict_proba', 'predict_log_proba']:
+        true_classes = np.unique(y)
+        train_classes = np.unique(y_train)
```

Comment:
But a few estimators implementing the `predict_proba` method, don't seem to have `classes_` as an attribute like `SVC`.